features/shop/apply_correct_simple_promotion_on_order.feature

```diff
@@ -0,0 +1,32 @@
+@ui-cart
+Feature: Apply correct simple promotions on order
+    In order to pay proper amount while buying promoted goods
+    As a Customer
+    I want to have promotions applied
+
+    Background:
+        Given the store is operating on a single channel
+        And default currency is "EUR"
+        And store has a product "PHP T-Shirt" priced at "€100.00"
+        And store has a product "Symfony Mug" priced at "€5.00"
+        And there is promotion "Special discount"
+        And it has "€10.00" "fixed discount" for cart with "item total" above "€80.00"
```

Comment:
This leaks implementation detail and does not sounds natural when you read it. I think we should have steps for different promotions, it is not that big overhead:

```
And store is running promotion "Christmas Sale"
And it gives €10 discount for customers with carts above €80
```